examples/audio/main/bsp_audio_example.c

```diff
@@ -249,9 +200,9 @@ void app_main(void)
     assert(ret == pdPASS);
 
     /* Init audio buttons */
-    for (int i = 0; i < (BSP_BUTTON_NUM - 1); i++) {
+    for (int i = 0; i < (BSP_BUTTON_NUM); i++) {
         audio_button[i] = iot_button_create(&bsp_button_config[i]);
         assert(audio_button[i] != NULL);
-        ESP_ERROR_CHECK(iot_button_register_cb(audio_button[i], BUTTON_PRESS_DOWN, btn_handler, NULL));
+        ESP_ERROR_CHECK(iot_button_register_cb(audio_button[i], BUTTON_PRESS_DOWN, btn_handler, (void *) & (bsp_button_config[i].adc_button_config.button_index)));
```

Comment:
After my work on LyraT, I would like to propose these changes in button intitialization:
```
if (bsp_button_config[i].type == BUTTON_TYPE_ADC) {
            ESP_ERROR_CHECK(iot_button_register_cb(audio_button[i], BUTTON_PRESS_DOWN, btn_handler, (void *) & (bsp_button_config[i].adc_button_config.button_index)));
        } else if (bsp_button_config[i].type == BUTTON_TYPE_GPIO) {
            ESP_ERROR_CHECK(iot_button_register_cb(audio_button[i], BUTTON_PRESS_DOWN, btn_handler, (void *) & (bsp_button_config[i].gpio_button_config.gpio_num)));
        } else if (bsp_button_config[i].type == BUTTON_TYPE_CUSTOM) {
            ESP_ERROR_CHECK(iot_button_register_cb(audio_button[i], BUTTON_PRESS_DOWN, btn_handler, (void *) & (bsp_button_config[i].custom_button_config.priv)));
        }
```